scripts/qa_crowbarsetup.sh

```diff
@@ -440,7 +440,11 @@ function add_sdk_repo
 function add_hcp_repo
 {
     # priority required to overwrite the default cloud6 packages
-    zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/HCP/SLE_12_SP1/Devel:Cloud:6:HPC.repo
+    if [ -z "$TESTHEAD" ]; then
+        zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/HCP/SLE_12_SP1/Devel:Cloud:6:HPC.repo
+    else
+        zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/Staging:/HCP/SLE_12_SP1/Devel:Cloud:6:HPC.repo
+    fi
```

Comment:
just an idea to reduce duplication:
```
local stage=
[[ $TESTHEAD ]] && stage="Staging:/"
zypper ar -p 92 -f http://$susedownload/ibs/Devel:/Cloud:/6:/${stage}HCP/SLE_12_SP1/Devel:Cloud:6:HPC.repo
```

Btw. is the name of the repo file always without staging?
If not then maybe drop the file name and pass a static name for the repo instead, eg. "dc6hpc".